netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -27,28 +90,72 @@ const uint32_t _net_ebpf_extension_sock_addr_bpf_attach_types[] = {
 
 #define NET_EBPF_SOCK_ADDR_HOOK_PROVIDER_COUNT EBPF_COUNT_OF(_net_ebpf_extension_sock_addr_attach_types)
 
-const net_ebpf_extension_wfp_filter_parameters_t _net_ebpf_extension_sock_addr_wfp_filter_parameters[] = {
+net_ebpf_extension_wfp_filter_parameters_t _cgroup_inet4_connect_filter_parameters[] = {
     {&FWPM_LAYER_ALE_AUTH_CONNECT_V4,
+     NULL,
```

Comment:
Either comment here or hard code default_sublayer. Easier to understand.

Reply:
Added comment